packages/dustkit/src/bridge/schemas.ts

```diff
@@ -61,4 +61,17 @@ export type ClientRpcSchema = RpcSchema.From<
         z: number;
       };
     }
+  | {
+      Request: {
+        method: "setBlueprint";
+        params: {
+          creationId: bigint;
```

Comment:
FYI - this schema will change a bit once I add orientation and remove the old cruft: ie creationId, displayName

(fine for now though)

Reply:
yea you can make the API match what the contract needs and makes sense

and then stub the values the current client needs, eg creationId